### PR TITLE
DM-40046: Fix unresolved merge conflict with gen3.sqlite3

### PR DESCRIPTION
**Pull request by erinleighh**

### CODE REVIEW

File: preloaded/gen3.sqlite3
Comment (kfindeisen) [inline preloaded/gen3.sqlite3:1]: Did you run `make_preloaded_export.py` after updating the registry? I'm expecting that, like on #45, the only changes would be the dataset IDs (which apparently don't interfere with import), but it's a good sanity check that the registry contains what we think it does.

**Review by kfindeisen — CHANGES_REQUESTED** [review]
I'm assuming that `ap_verify` runs with this version of the registry, but I'd feel better if the `export.yaml` were re-synced so we could see what's inside.

File: preloaded/gen3.sqlite3
Comment (erinleighh) [reply preloaded/gen3.sqlite3:1]: Yes, I ran `make_preloaded_export.py` after updating, which didn't have any issues. I'll re-sync the `export.yaml`.

Comment (kfindeisen) [reply preloaded/gen3.sqlite3:1]: From Slack discussion, it turns out that the registry really is identical down to the dataset IDs, so there's nothing to push.

**Review by kfindeisen — APPROVED** [review]

**Pull request merged**